src/Pages/FormSliderPage.tsx

```diff
@@ -90,7 +92,7 @@ const FormSliderPage = () => {
           />
         </PagePaper>
       </Box>
-      <TableOfContent />
+      {!isSmallScreen && <TableOfContent />}
```

Comment:
Let's just create a wrapper component that already has this repetitive code on all pages.